app/src/main/kotlin/com/simplemobiletools/dialer/helpers/RecentsHelper.kt

```diff
@@ -44,7 +45,10 @@ class RecentsHelper(private val context: Context) {
         val contactsNumbersMap = HashMap<String, String>()
         val contactPhotosMap = HashMap<String, String>()
 
-        val uri = Calls.CONTENT_URI
+        val uri = Calls.CONTENT_URI.buildUpon()
+            .appendQueryParameter(Calls.LIMIT_PARAM_KEY, QUERY_LIMIT)
```

Comment:
so this works well on Android 6 too?

Reply:
just fixed that in https://github.com/SimpleMobileTools/Simple-Dialer/pull/379/commits/d6025e59624641f016412e08bdbb6c7a64b11512